GRPCServiceA/pkg/utils/queries.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+var GetQueryByEmail string = "SELECT id, name, age, email, nationality, job, created FROM Users WHERE email=?"
```

Comment:
this is not variable in the logic, can be Cont var and unify declarations 
const ( 
GetByEmail = ....
GetById = ...
)